### DISCUSSION TITLE
Multireddit and saved posts on logged out/anonymous/accountless/lurker mode

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#967] [Ideas]
It would be great if multireddit and saved posts can be implemented to work with logged out/anonymous/accountless/lurker mode

**Comment by KAGEYAM4** [comment]
@Docile-Alligator  , i looked past issues for feature request to save post in anonymous , but it was not clear if it will be implemented or won't . Can you please tell if you're interested in this or will be doing it it future ?

**Comment by Docile-Alligator** [comment]
@PhoenixCODE2322 This will be implemented. The reason why I haven't done it is because I couldn't find a way to fetch all the post info by providing only the id of the posts. But now I understand how it works so this feature will be implemented.

**Reply by AnantTiwari-Naman** [reply]
do you still have plans to add this to Infinity? it would help me out a lot

**Reply by kpupp** [reply]
I'd also love if this could be implemented. Wouldn't mind the storage costs associated, and would also love to not have to log in all the time.

**Comment by mangkoran** [comment]
I saw this discussion and it seems this is related to https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/1391. I would also like to see the multireddit on anon mode feature.

**Reply by kpupp** [reply]
You can create multireddits on infinity without logging in, if that's what you're looking for. I think all that's left is saving posts.

**Reply by mangkoran** [reply]
Oh sorry I think I didn't include the correct context. What I would like to is adding other's / public multireddit instead of creating a new one (e.g. multireddit in r/multihub). Is it also possible?

**Comment by kpupp** [comment]
@Docile-Alligator is this still planned? It's really the one thing keeping me from using Infinity more often (now that reddit's bots had their way with my account(s)).
